File: DAW/UF1/1_demo_llibreria_estatica/LlibreriaCrypto/Crypto.cpp
#include "Crypto.h"

void encripta(char *msg) {
	while ((*msg) != 0) {
		(*msg) += 15;
		msg++;
	}
}

void desencripta(char *msg) {
	while ((*msg) != 0) {
		(*msg) -= 15;
		msg++;
	}
}

File: DAM/UF1/4_ExempleCreacioLlibreriaNetbeans/LlibreriaCriptografica/src/com/thecrypt/Crypto.java
package com.thecrypt;

/**
 *
 * @author Usuari
 */
public class Crypto {

    public String encripta(String original) {
        String out="";
        for(int i=0;i<original.length();i++) {
            out += (char)(original.charAt(i)-15);
        }
        return out;
    }
    public String desencripta(String original) {
        String out="";
        for(int i=0;i<original.length();i++) {
            out += (char)(original.charAt(i)+15);
        }
        return out;
    }
    //----------------------------------------
    public static void main(String[] args) {
        Crypto c = new Crypto();
        String encriptada = c.encripta("Hola Món");
        System.out.println(encriptada);
        System.out.println(c.desencripta(encriptada));        
    }
    
    
}
